sphinx_jupyterbook_latex/__init__.py

```diff
@@ -15,10 +15,15 @@ def setup(app: "Sphinx") -> None:
 
     from .events import override_latex_config, setup_latex_transforms
     from .nodes import HiddenCellNode, RootHeader
-    from .transforms import LatexRootDocTransforms
+    from .transforms import (
+        LatexRootDocTransforms,
+        LatexToctreeNodeInterpret,
+        SwapTableofContents,
+    )
 
     # autoload the sphinx.ext.imgconverter extension
     app.add_config_value("jblatex_load_imgconverter", True, "env")
+    app.add_config_value("jblatex_show_tocs", "list", "env", (str, bool))
```

Comment:
@AakashGfude should the default be `True` here as then if we change the default from "list" to something else this won't need to be changed right.

Reply:
done in https://github.com/executablebooks/sphinx-jupyterbook-latex/pull/64/commits/ba3b77d279ea6dc572d4dcfb818a2a54b2f08f6b